etc/ci/performance/test_differ.py

```diff
@@ -49,4 +53,13 @@ def load_data(filename):
         diff = value2 - value1
         change = diff / value1
         color = BLUE if value1 <= value2 else GREEN
+        total1 += value1
+        total2 += value2
         print("{}{:6} {:6} {:+6} {:+8.2%}   {}.{}".format(color, value1, value2, diff, change, key, END))
+
+
+print()
+diff = total2 - total1
+change = diff / total1
+color = BLUE if total1 <= total2 else GREEN
+print("{}{:6} {:6} {:+6} {:+8.2%}   {}.{}".format(color, total1, total2, diff, change, "TOTAL", END))
```

Comment:
It might be nice to pull these 4 lines into a helper.

Reply:
Fixed.